packages/opentelemetry-plugin-grpc/src/grpc.ts

```diff
@@ -617,6 +589,19 @@ export class GrpcPlugin extends BasePlugin<grpc> {
     }
     return metadata;
   }
+
+  /**
+   * Convert a grpc status code to an opentelemetry Canonical code. For now, the enums are exactly the same
+   * @param status
+   */
+  private static _grpcStatusCodeToCanonicalCode(
```

Comment:
Rather than making this `private static`, what would you think about just having it be a non-exported `function` in the file? I personally find that cleaner for helper functions than attaching them to a class

Reply:
+1

Reply:
I have an existing `utils.ts` already. Maybe I can just move them inside there?